srcpkgs/kodi/template

```diff
@@ -110,8 +110,10 @@ post_build() {
 	fi
 }
 
-post_install() {
-	# remove unused stuff
-	rm -rf ${DESTDIR}/usr/include
-	find ${DESTDIR}/usr/lib -name "*.cmake" -delete
```

Comment:
Are the .cmake now nonexistant? Since they normally should also be moved to the devel subpackage